azerothcore/azerothcore-wotlk
# feat(Core/AI): CU_SAI - Option to set creature in combat with zone outside of a dungeon

## CHANGES PROPOSED:
-  It is now possible to use the action SMART_ACTION_SET_IN_COMBAT_WITH_ZONE outside of a dungeon by with using a range


## TESTS PERFORMED:
- Was on a 2k player live server.


## HOW TO TEST THE CHANGES:

- Create a simple SAI with action 38 and enter a range as first param
- Spawn it outside a dungeon (open world)
- Check if it goes IC with you and not the whole map ;-)

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
